entrypoint.sh

```diff
@@ -14,6 +14,9 @@ if [ "$1" == "install" ]; then
     echo "  run:     docker run --rm --pid=host aquasec/kube-bench [command]"
     exit
   fi
+elif [ "$1" == "repeat" ]; then
+  echo "Now scheduling kube-bench to run every 24 hours"
+  ./repeat-loop.sh &
```

Comment:
Running this in the background means that this entrypoint.sh script can exit...which then terminates the container